db/migrate/20220430105035_create_posts.rb

```diff
@@ -0,0 +1,8 @@
+class CreatePosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+
+      t.timestamps
+    end
```

Comment:
- [ ]  There is no migration column added. Make sure you have proper migration created for post.
![image](https://user-images.githubusercontent.com/58504655/166104251-fbf68282-6d7d-40d1-bcfa-9276417cc1b1.png)

Reply:
thanks for the feedback @arslanbisharat but the migrations to `add_text_to_post`,`add_title_to_post` `add_comment_counter_to_post` and `add_comment_counter_to_post` are present in separate

Reply:
I can see you have created individual migration for each property but this makes the database hard to read and manage. Please try to follow the comments given below.